orangecontrib/bioinformatics/widgets/OWMarkerGenes.py

```diff
@@ -84,12 +84,14 @@ def __init__(self, data, parent):
             partial(defaultdict, partial(defaultdict, lambda: None)),
         )(self._roleData)
         self.set_column_links()
+        self.source = data
```

Comment:
Why is this necessary (source attribute should only be set by wrap method in SearchableTableModel)?
I think this can be safely removed.

Reply:
I removed this line in last commit.